pkg/datapath/loader/loader.go

```diff
@@ -440,8 +406,17 @@ func (l *loader) reloadHostDatapath(ctx context.Context, ep datapath.Endpoint, s
 			return err
 		}
 
-		progs := []progDefinition{
-			{progName: symbolFromHostNetdevEp, direction: dirIngress},
+		// Load bpf_netdev_<dev>.o.
```

Comment:
Kind of misleading, now that this file doesn't exist anymore?

Reply:
Thanks, good spot. Took the opportunity to document entrypoint names + hook points in the function doc as well. Didn't realize this function only deals with bpf_host, we should make that more explicit through means of API.